src/Symfony/Component/Serializer/DependencyInjection/SerializerPass.php

```diff
@@ -50,11 +50,39 @@ public function process(ContainerBuilder $container)
 
         $serializerDefinition = $container->getDefinition($this->serializerService);
         $serializerDefinition->replaceArgument(0, $normalizers);
+        $this->addDefaultContextParameterBinding($normalizers, $container);
 
         if (!$encoders = $this->findAndSortTaggedServices($this->encoderTag, $container)) {
             throw new RuntimeException(sprintf('You must tag at least one service as "%s" to use the "%s" service.', $this->encoderTag, $this->serializerService));
         }
 
         $serializerDefinition->replaceArgument(1, $encoders);
+        $this->addDefaultContextParameterBinding($encoders, $container);
+    }
+
+    private function addDefaultContextParameterBinding($services, $container)
```

Comment:
```suggestion
    private function addDefaultContextParameter($services, $container)
```

Binding is confusing, let's remove it